master/master.cfg

```diff
@@ -65,12 +69,31 @@ from custom.workers import get_workers  # noqa: E402
 settings_path = os.path.join(os.path.dirname(__file__), 'settings.yaml')
 try:
     settings = Settings.from_file(settings_path)
+    try:
+        release_sha = subprocess.check_output(["git", "rev-parse", "HEAD"], text=True)
+        sentry_sdk.init(dsn=settings.sentry_dns, release=release_sha,
```

Comment:
`dsn` or `dns`?

This should probably use `settings.get('sentry_dns', None)` and check for `None`.

Reply:
Is DNS (Data Source Name). You can check in https://docs.sentry.io/error-reporting/quickstart/?platform=python